content/history/org.inc

```diff
@@ -99,7 +99,7 @@ while($r = $recordData->fetch_assoc()) {
 // And after that convoluted query, we have an absolutely enormous query to get all the historical team times.
 //
 // In an unusual step, we don't calculate the pretty version of time in the SQL query, because we need to hand the
-// graphing tool raw seconds.
+// graphing tool raw seconds, so we just use our displayTime() utility function.
```

Comment:
Oooh would it be worthwhile killing time formatting from the queries generally and using `displayTime` instead?

Reply:
TLDR: Yes, probably.

I was pretty far in using the sample queries Jeremy provided when I saw the 1 place (graphing) where I actually needed the seconds, and the one other place where it was basically untenable because of the query complexity.  So I kept going with the formatting unless I couldn't.

But though I don't think it matters performance wise at all (our DB and app server are the same, and frankly we have trivial amount of data overall), it certainly makes the queries harder to read in the code (if easier to use in debugging), so I'll look at taking it out.

Reply:
Change made.

Reply:
Looks beautiful, thank you so much!